crispy_bootstrap5/templates/bootstrap5/layout/formactions.html

```diff
@@ -1,6 +1,6 @@
 <div
     {% if formactions.flat_attrs %}{{ formactions.flat_attrs }}{% endif %} 
-    class="mb-3{% if 'form-horizontal' in form_class %} row{% endif %} {{ formactions.css_class|default:'' }} {{ field_class }}" 
```

Comment:
Removing this doesn't look like the correct fix to me. `field_class` can be set by users and I expect it to have no impact by default. 

Could you explain why this was impacting your example?

Reply:
In the original FormAction template `field_class` was added twice: to the wrapper div around label and the inner div and the inner div itself.

And I suspected that assigning this class twice was added by mistake.

Reply:
For example this template has only on occurrence of `{{field_class}}` https://github.com/django-crispy-forms/crispy-bootstrap5/blob/main/crispy_bootstrap5%2Ftemplates%2Fbootstrap5%2Flayout%2Funeditable_input.html